### PR TITLE
Format: break long lines as requested in review of #14

### PR DESCRIPTION
**Pull request by choeppler**
In the review of #14 there was a request to break long lines to ease
reading in some text editors. This PR does that.

### CODE REVIEW

File: design/003-IoT-Adoption.md [hunk 1]
```diff
@@ -65,6 +113,8 @@ Architecture
 
 [1] Dziembowski, Faust, Hostáková (2018): General State Channel Networks
 
-[2] Faust, Dziembowski, Eckey, Malinowski (2018): Perun: Virtual Payment Hubs over Cryptocurrencies
+[2] Faust, Dziembowski, Eckey, Malinowski (2018): Perun: Virtual Payment Hubs
+over Cryptocurrencies
 
-[3] McCorry, Bakshi, Bentov (2018): Pisa: Arbitration Outsourcing for State Channels
+[3] McCorry, Bakshi, Bentov (2018): Pisa: Arbitration Outsourcing for State
+Channels
```
Comment (manoranjith) [inline design/003-IoT-Adoption.md:120]:
```suggestion
[3] McCorry, Bakshi, Bentov (2018): Pisa: Arbitration Outsourcing for State
    Channels
```

File: design/003-IoT-Adoption.md [hunk 2]
```diff
@@ -65,6 +113,8 @@ Architecture
 
 [1] Dziembowski, Faust, Hostáková (2018): General State Channel Networks
 
-[2] Faust, Dziembowski, Eckey, Malinowski (2018): Perun: Virtual Payment Hubs over Cryptocurrencies
+[2] Faust, Dziembowski, Eckey, Malinowski (2018): Perun: Virtual Payment Hubs
+over Cryptocurrencies
```
Comment (manoranjith) [inline design/003-IoT-Adoption.md:117]:
```suggestion
[2] Faust, Dziembowski, Eckey, Malinowski (2018): Perun: Virtual Payment Hubs
    over Cryptocurrencies
```

**Review by manoranjith — CHANGES_REQUESTED** [review]
LGTM.

I have one minor suggestion: adding a few spaces to align the text for bullet points (in references).

File: design/003-IoT-Adoption.md (hunk 1 above)
Comment (choeppler) [reply design/003-IoT-Adoption.md]: Done in 80360ae4

File: design/003-IoT-Adoption.md (hunk 2 above)
Comment (choeppler) [reply design/003-IoT-Adoption.md]: Done in 80360ae4

**Review by manoranjith — APPROVED** [review]
LGTM.

**Pull request merged**